src/store/cache/music/hooks.ts

```diff
@@ -88,6 +90,7 @@ export function fetchTopAlbums(
         handle: d.user.handle,
         artwork: d.artwork?.['480x480'] ?? imageBlank,
         plays: d.total_play_count,
+        favorites: d.favorite_count,
```

Comment:
Do we care about trackCount here too?

Reply:
we actually don't. i removed all of these vars. we decided better to just not show anything